saleor/product/migrations/0187_auto_20230911_0704.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.20 on 2023-09-11 07:04
+
+import django.contrib.postgres.indexes
+from django.db import migrations, models
+from django.contrib.postgres.operations import AddIndexConcurrently
+
+
+class Migration(migrations.Migration):
+    atomic = False
+
+    dependencies = [
+        ("product", "0186_remove_product_charge_taxes"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="category",
+            name="updated_at",
+            field=models.DateTimeField(auto_now=True, null=True),
+        ),
+        AddIndexConcurrently(
+            model_name="category",
+            index=django.contrib.postgres.indexes.GinIndex(
```

Comment:
`GinIndex` will not be a good index to use for that field. I think default one (which is B-Tree index) should be good here.

Reply:
@kadewu I have also changed the index.